### DISCUSSION TITLE
Podcast Autodownload

### DISCUSSION THREAD

**Discussion by truthsword** [advplyr/audiobookshelf-app#896] [Ideas]
I'd like to see the option to auto download available podcasts, and also to be able to restrict auto-downloads to Wi-Fi connections. Great app! Thanks for all your work.

**Comment by nichwall** [comment]
This has been mentioned a few times here and on Discord. Two of the issues are https://github.com/advplyr/audiobookshelf-app/issues/774 and https://github.com/advplyr/audiobookshelf-app/issues/247

**Comment by truthsword** [comment]
I'm more interested in podcast downloading as they are consumed much more rapidly than books. Hopefully this might come about in a future release.

**Reply by politas** [reply]
Yep, we should have a setting per playlist to automatically download items.

**Comment by cameronj86** [comment]
Good to see others also want the feature. 

The average audiobook is ~10 hrs long, so auto-download are understandably a lower priority since it happens so infrequently. Conversely, podcasts are much shorter and more frequent, so auto-downloading would be a much more appreciated setting for those users.

**Comment by truthsword** [comment]
I switched back to Pocket Casts... Too many times on the road, where I discovered I forgot to manually download podcasts before leaving the house (I'm data constrained). Pocket Cast does that for me automagically over Wi-Fi..

**Comment by pippo73** [comment]
If it could help, I add myself to the list of people interested in this feature :-)
